public/static/locales/de/common.json

```diff
@@ -46,6 +46,8 @@
   "continueToLogin": "Weiter zum Login",
   "supportsUNEP": "Unterstützt das UNEP",
   "plantedBy": "Gepflanzt von",
-  "mapInfo": "Esri Info",
-  "aboutUs": "Über uns"
-}
\ No newline at end of file
+  "mapInfo": "Esri",
+  "aboutUs": "Über uns",
+  "redeemPopup": "Baumgeschenk Bekommen? Log dich ein um den Code Einzulösen",
```

Comment:
This need to be fixed (in LingoHu)b: "Baumgeschenk bekommen? Log dich ein, um den Code einzulösen!".